tests.py

```diff
@@ -196,3 +203,26 @@ def test_valid_hook_module(tmpdir, monkeypatch):
     output = result.output.splitlines()
     assert output[0] == "from on_pre_build"
     assert output[-1] == "from on_post_build"
+
+
+def test_disabling_plugin(tmpdir, monkeypatch):
+    with open(str(tmpdir / "hooks.py"), "w") as f:
+        f.write(
+            "def on_pre_build(*args, **kwargs):\n"
+            '    print("from on_pre_build")\n'
+            "def on_post_build(*args, **kwargs):\n"
+            '    print("from on_post_build")\n'
+        )
+
+    runner = setup_mkdocs(
+        {"on_pre_build": "hooks:on_pre_build", "on_post_build": "hooks:on_post_build",},
+        monkeypatch,
+        tmpdir,
+        enabled=False,
+    )
+
+    result = runner.invoke(build_command)
+    assert result.exit_code == 0
+    output = result.output.splitlines()
+    assert output[0] != "from on_pre_build"
+    assert output[-1] != "from on_post_build"
```

Comment:
I think the `output` should be a string, and the assertions can just check that the printed messages are not in `output`. In theory, if these prints would exist but switch positions, this test would still pass. 
Also, it would be enough to have just one hook defined with a single print. Just its existence would indicate if the plugin is enabled or not.

Reply:
Fixed. Btw, I had just copied the test above it, which uses the same syntax.

Reply:
Yes, I know, but for these tests, the order of the messages is actually important, that's why it's done this way.